client/me/developer/main.jsx

```diff
@@ -67,8 +163,13 @@ export default compose(
 	connect(
 		( state ) => ( {
 			isFetchingUserSettings: isFetchingUserSettings( state ),
+			currentUser: getCurrentUser( state ),
 		} ),
-		{ recordGoogleEvent }
+		{
+			recordGoogleEvent,
```

Comment:
```suggestion
```
This was probably added by me by mistake when I first created the page :( `recordGoogleEvent` isn't being used, we can remove it.